package/toltec-meta/package

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+archs=(rm1 rm2)
+pkgnames=(toltec-meta)
+pkgdesc="Toltec meta packages to ensure that the required base packages get installed"
+url=https://toltec-dev.org/
+pkgver=1.0-1
+timestamp=2022-02-28T00:12Z
+section="utils"
+maintainer="Eeems <eeems@eeems.email>"
+license=MIT
+installdepends=(toltec-bootstrap toltec-completion xochitl)
```

Comment:
If we’re changing `xochitl` to be installed by default, would it make sense to remove it from the installdepends arrays of `display`, `draft`, `reboot-guard`, `tarnish`, and `remarkable-stylus`?

Reply:
No, even if something is installed by default. They should still be marked as depends by anything that uses them. That way if you try to uninstall a default package you get proper dependency warnings, and can get it back automatically if you install something that uses it again. Being explicit is not a bad thing here.

Reply:
That’s a good point. I don’t think we should have it installed by default.